tests/gatling_kickoff.rb

```diff
@@ -1,6 +1,26 @@
-step "Execute gatling scenario" do
-  on metric, "cd /root/gatling-puppet-load-test/simulation-runner/ && " +
-      "PUPPET_GATLING_MASTER_BASE_URL=https://#{master.hostname}:8140 " +
-      "PUPPET_GATLING_SIMULATION_CONFIG=config/scenarios/ApplesToApples.json " +
-      "PUPPET_GATLING_SIMULATION_ID=PerfTestLarge sbt run"
+require 'beaker-benchmark'
+
+test_name 'Kickoff Gatling' do
+
+  atop_log = '/tmp/atop.log'
+  start_monitoring(master, atop_log, true)
+
+  begin
+    step "Execute gatling scenario" do
+      on metric, "cd /root/gatling-puppet-load-test/simulation-runner/ && " +
+          "PUPPET_GATLING_MASTER_BASE_URL=https://#{master.hostname}:8140 " +
+          "PUPPET_GATLING_SIMULATION_CONFIG=config/scenarios/ApplesToApples.json " +
+          "PUPPET_GATLING_SIMULATION_ID=PerfTestLarge sbt run",
+         {:accept_all_exit_codes => true}
+    end
+
+  ensure
+    perf_result = stop_monitoring(master, 'Apples to Apples gatling run', atop_log, '/opt/puppetlabs')
```

Comment:
internally stop monitoring looks odd to me...
I see it do this
scp_from(infrastructure_host, atop_log_name, '/tmp')
followed by this
CSV.parse(File.read("/tmp/#{File.basename(atop_log_name)}"), { :col_sep => ' ' }) do |row|

The first looks like it would create /tmp/tmp/atop.log on the testrunner.
The second looks like it would open /tmp/atop.log to parse.
That seems like it should fail.